axiom/nr_php_packages.c

```diff
@@ -116,6 +121,16 @@ nr_php_package_t* nr_php_packages_add_package(nr_php_packages_t* h,
   return p;
 }
 
+void nr_php_packages_iterate(nr_php_packages_t* packages,
+                             nr_php_packages_iter_t callback,
+                             void* userdata) {
+  if (NULL == packages || NULL == callback) {
```

Comment:
Eventual check for `NULL == userdata` happens in `callback`, yes?

Reply:
Ok, I see it happens [here](https://github.com/newrelic/newrelic-php-agent/blob/4ea22b6bc5d62439e690a5e2885e318a0b9b1a84/agent/php_txn.c#L727-L729).